README.md

```diff
@@ -303,6 +306,18 @@ Specifies the tenant ID of the the tenant the Azure SQL database exists in. This
 		AzureTenantId = Environment.GetEnvironmentVariable("AZURE_TENANT_ID")
 	}
 ```
+### RetentionPeriod
+
+Specifies a period, till now, in which the non-audit sink logs would be retained. For example, when its value is 6 months. That means in every cleaning, the logs older than 6 months would be cleaned.
+
+### CleaningInterval
+
+Specifies the interval in which the non-audit sink is being clean of old logs. It's default value is 2 hours. The `old` state would be determined by the `RetentionPeriod`.
+
+_NOTE:_ The cleaning only occurs in the batch write events, Therefore if the interval is satisfied the actual cleaning would be triggered in the next batch write event.
+
+_NOTE:_ If the `RetentionPeriod` is not set, The `CleaningInterval` would never be checked. And logs would never be touched for the cleaning.
```

Comment:
Please add a note regarding performance. DbCommand is slower than BulkInsert. Athough okay when it only happens every 2 hours or so.

Reply:
These are tests I have executed, each of which took around 10 minutes.
| Log Number | Each Prune Slowdown Percentage| Total Slowdown Percentage | Pruning Number |Pruning Interval (ms) | Approx Expected Retention Size |Retention Period (ms)|
| ----------- | ----------- |----------- | ----------- |----------- | ----------- | ----------- |
| 4000000| 0.01156783345|0.1260234846| 10.89430316|30000| 390000|30000|
| 4000000| -0.05064482868|-0.5559083096| 10.97660559|30000| 780000|60000|
| 4000000|0.03845117802|0.4237652983| 10.97660559|30000| 1170000|90000|
| 4000000| -0.01017271449|-0.1116879975| 10.80066054|30000| 1560000|120000|
| 4000000|-0.04309239407|-0.4654263203| 10.97660559|30000| 1950000|150000|
| 4000000|0.08813716002|0.9582288517|10.8720187|30000| 2340000|180000|
| 4000000| 0.0117173788|0.748410463|63.87183308|5000| 390000|30000|
| 4000000|-0.009383170383|-0.5935576968| 63.25769144|5000| 780000|60000|
| 4000000|-0.001097505776|-0.06988459624| 63.67583456|5000| 1170000|90000|